src/main/java/com/twilio/verification/logging/LoggingFilter.java

```diff
@@ -0,0 +1,153 @@
+package com.twilio.verification.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
```

Comment:
@hortega quick question, is it needed to import the two lines above even if you have already used `java.util.*`?